llvm/utils/TableGen/VTEmitter.cpp

```diff
@@ -79,12 +79,12 @@ static void VTtoGetLLVMTyString(raw_ostream &OS, const Record *VT) {
 void VTEmitter::run(raw_ostream &OS) {
   emitSourceFileHeader("ValueTypes Source Fragment", OS, Records);
 
-  std::array<const Record *, 256> VTsByNumber = {};
+  std::array<const Record *, 65536> VTsByNumber = {};
```

Comment:
16384? And should probably change to std::vector to avoid a large stack allocation.

Reply:
Sure~